### ISSUE TITLE
Custom aliases have stopped appearing

### ISSUE THREAD

**Issue by VictorSabido** [rioukkevin/vscode-git-commit#11]
Hi, 
I think my customs labels have **stopped appearing** since the **last update**, what am I doing wrong? Now only the default ones appear. I don't know what the "keys" are either.
I have tried to change the "prefix" and it doesn't work either 

Thanks in advance

![imagen](https://user-images.githubusercontent.com/19414594/115549103-8d137480-a2a8-11eb-8f2d-bf951793d10d.png)

**Comment by rioukkevin** [comment]
Sorry, I work on it, I forgot the merging abilities

While i'm working on you can set `vscodeGitCommit.variables.prefix` to 
```
[
  // Add your aliases here
  {
    label: '✨feature',
    detail: 'Select when creating new things',
    key: 'feature',
  },
  {
    label: '🐞fix',
    detail: 'Select when fixing a bug',
    key: 'fix',
  },
  {
    label: '📄docs',
    detail: 'Select when editing documentation',
    key: 'docs',
  },
  {
    label: '🖥️wip',
    detail: 'Select when work is not finished',
    key: 'wip',
  },
  {
    label: '🚅perfs',
    detail: 'Select when working on performances',
    key: 'perfs',
  },
  {
    label: '⏪rollback',
    detail: 'Select when undoing something',
    key: 'rollback',
  },
  {
    label: '🔵other',
    detail: 'Select when fixing a bug',
    key: 'other',
  },
]
```
And add your aliases at the comment `// Add your aliases here`

Sorry for the inconvenience 😞

**Comment by VictorSabido** [comment]
It didn't work for me. I look forward to the next version! Thanks

**Comment by rioukkevin** [comment]
I fix it in the new release (2.1.0) you can change your prefix variable with content `aliases...keke` it will merge aliases variable setting and the preset keke
